src/style.css

```diff
@@ -1 +1,87 @@
-/* add styles here */
+:root {
+  --color-light: #e1e7ed;
+  --color-dark: #3d4e61;
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url("fonts/Avenir-Heavy.ttf");
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.form {
+  margin-top: 20px;
+  position: relative;
+}
+
+.form::after {
+  position: absolute;
+  content: "";
+  display: block;
+}
+
+.form_big::after {
+  background-image: url("./images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.form_small::after {
+  background-image: url("./images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.form__input {
+  width: 100%;
+  border: 1px solid var(--color-light);
+  border-radius: 4px;
+  color: var(--color-dark);
+  font-family: Avenir, sans-serif;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)